src/bootloaders/mbr.c

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Oof I don't really like this as we now need to keep looking to see if this changes whenever a distro using clr-boot-manager updates syslinux. This just seems like a bit of a nightmare for distro maintenance.

Reply:
This is actually easier. The latest upstream stable release MBR's should always be OK to use, even when they differ with a distro's copy of them. The location of these files can vary wildly. On top of that, We should *still* embed these in case we fail to find the MBR copy on the rootfs.

So this file should really stay. If we want to prefer the local copy (I personally don't, but I could be OK with it) - then we should still have a fallback.

Reply:
I'd say patching the path is easier than patching this binary file when the paths differ from the default. Perhaps it isn't very likely that there is a problem with the file but us having to keep checking for releases in the upstream and updating the binary in cbm isn't something I'm eager to do especially with syslinux in a weird place with their release/tagging/maintenance.

I'm not NAKing the patch though so if that's the way we want to maintain it I'll go along with it. The change has a lot of benefits for as much as I don't like some of its implications.

Reply:
I'm more inclined to have it loading from the distributed binary, we can implement a set of fallbacks of  common installed paths.

Reply:
I've seen these files missing entirely. We should really have a backup handy. Being able to recover from an unbootable system with just `/usr/bin/syslinux` is a wonderful feature and worth the 3kb text.